src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -3753,7 +3849,7 @@
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
-				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG UITESTING";
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG UITESTING INTEROP";
```

Comment:
Interpo should be disabled on dev.

Reply:
This is for UI tests right? I would rather test the ui with the new features enabled as they would appear soon™. Unfortunately we can't do both without major refactorings.

Reply:
Ah you are right, sorry just saw INTEROP and my alarm got trigged :D

Reply:
Actually I was hoping to see such a comment. Good catch 👍